spec/ui/command_line_spec.moon

```diff
@@ -0,0 +1,276 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+import app, dispatch, interact from howl
+import Window from howl.ui
+
+describe 'commandline', ->
+  local command_line, within_activity
```

Comment:
The `within_activity` shadows the global helper here, and does something else it seems. Rename to something else?